.github/workflows/build.yml

```diff
@@ -23,15 +23,21 @@ jobs:
         with:
           submodules: recursive
       - name: Install dependencies
-        run: brew install ldid
+        run: |
+          brew install ldid
+          curl https://sh.rustup.rs -sSf | sh -s -- -y
+          rustup target add aarch64-apple-ios
+
       - name: Setup Xcode
         uses: maxim-lobanov/setup-xcode@v1.4.1
         with:
           xcode-version: ${{ matrix.version }}
       - name: Build SideStore
         run: |
+          export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
```

Comment:
What's this do?

Reply:
Makes sure that the environment uses the proper sdk without using linking as some of the items it would look in a different area and not find compileC. It's not particularly important I don't think but it did help during making this actions adjustment at the moment.